tests/e2e/test_e2e_template.py

```diff
@@ -68,6 +69,9 @@ def test_make_help_works() -> None:
 
 @pytest.mark.run(order=1)
 def test_make_setup() -> None:
+    if EXISTING_PROJECT_SLUG:
+        return
```

Comment:
Minor: better `@pytest.mark.skipif(EXISTING_PROJECT_SLUG)` so that we don't have this test green if skipped